gravitee-apim-console-webui/src/management/settings/notification/environment-notification.module.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatIconModule } from '@angular/material/icon';
+import { MatLegacySnackBarModule } from '@angular/material/legacy-snack-bar';
+
+import { EnvironmentNotificationComponent } from './environment-notification.component';
+
+import { NotificationModule } from '../../../components/notification';
+
+@NgModule({
+  declarations: [EnvironmentNotificationComponent],
+  exports: [],
+  imports: [CommonModule, MatCardModule, MatButtonModule, MatIconModule, MatDialogModule, NotificationModule, MatLegacySnackBarModule],
```

Comment:
We don't need MatLegacySnackBarModule anymore. i think you can juste remove it here